### PR TITLE
Display message when there is an empty restaurant list

### LINKED ISSUE #23: Handle Error when API returns empty restaurant list
- [X] Read status to see if there is an empty set
- [X] Display message prompting user to enter another address

### PR DESCRIPTION
**Pull request by WinsonChen114**

### CODE REVIEW

**Comment by WinsonChen114** [comment]
Related PR: https://github.com/SITE-2022-W6/The-Foodie-Express-API/pull/44

File: src/services/apiClient.jsx [hunk 1]
```diff
@@ -65,6 +65,7 @@ class ApiClient {
   {
     // console.log(cityState)
     let response = await this.request({endpoint: `restaurant/location/${cityState.state}/${cityState.city}/${offset}`})
+    console.log(response)
```
Comment (LifeBringer) [inline src/services/apiClient.jsx]: Comment out or delete

**Review by LifeBringer — CHANGES_REQUESTED** [review]
See comment

File: src/services/apiClient.jsx (hunk 1 above)
Comment (WinsonChen114) [reply src/services/apiClient.jsx]: Done

**Review by LifeBringer — APPROVED** [review]

**Pull request merged**